routers/api/v1/repo/pull.go

```diff
@@ -420,8 +421,46 @@ func CreatePullRequest(ctx *context.APIContext) {
 			return
 		}
 	}
+	// handle reviewers
+	var reviewerIds []int64
 
-	if err := pull_service.NewPullRequest(ctx, repo, prIssue, labelIDs, []string{}, pr, assigneeIDs); err != nil {
+	for _, r := range form.Reviewers {
+		var reviewer *user_model.User
+		if strings.Contains(r, "@") {
+			reviewer, err = user_model.GetUserByEmail(ctx, r)
+		} else {
+			reviewer, err = user_model.GetUserByName(ctx, r)
+		}
+
+		if err != nil {
+			if user_model.IsErrUserNotExist(err) {
+				ctx.NotFound("UserNotExist", fmt.Sprintf("User with id '%s' not exist", r))
+				return
+			}
+			ctx.Error(http.StatusInternalServerError, "GetUser", err)
+			return
+		}
+		reviewerIds = append(reviewerIds, reviewer.ID)
+	}
+
+	// handle teams as reviewers
+	if ctx.Repo.Repository.Owner.IsOrganization() && len(form.TeamReviewers) > 0 {
+		for _, t := range form.TeamReviewers {
+			var teamReviewer *organization.Team
+			teamReviewer, err = organization.GetTeam(ctx, ctx.Repo.Owner.ID, t)
+			if err != nil {
+				if organization.IsErrTeamNotExist(err) {
+					ctx.NotFound("TeamNotExist", fmt.Sprintf("Team '%s' not exist", t))
+					return
+				}
+				ctx.Error(http.StatusInternalServerError, "ReviewRequest", err)
+				return
+			}
+			reviewerIds = append(reviewerIds, teamReviewer.ID)
```

Comment:
Can we mix the user id an team id? I don't think so.

Reply:
Thanks i'm currently testing the change.

Reply:
Resolved this. Thank you!